main.tf

```diff
@@ -1,11 +1,11 @@
 # create test resource group
 resource "azurerm_resource_group" "test" {
-  name     = "test-rg"
+  name     = "test-rg123"
   location = "East US"
 }
 
-module "mysql_singleserver" {
-  source = "./modules/mysqlserver"
```

Comment:
Why did you replace here . Keep the old code of calling single server as well .. when you run this plan both should be created !